enos/modules/test_cluster_health/scripts/wait_for_nomad_api.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Doesn't this script already exist in the `run_workloads` module? Can these be reusable by placing them in the `enos/` root somewhere?

Reply:
that same script is used on multiple places, Im debating myself on reusing it or leave every module have its own thing

Reply:
Tell you what, for now we can just keep this separate and as we iterate we can tighten it up if possible.